### PR TITLE
Add lucasfcosta to contributors on README.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lucasfcosta**
Thanks for [the invite](https://twitter.com/Keithamus/status/708412140781223937) @keithamus!
Let's make great software hahahha
:tada: :tada: :tada: :tada:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ concerns. We will do our best to respond in a timely manner.
 [![Jake Luer](https://avatars3.githubusercontent.com/u/58988?v=3&s=50)](https://github.com/logicalparadox)
 [![Veselin Todorov](https://avatars3.githubusercontent.com/u/330048?v=3&s=50)](https://github.com/vesln)
 [![Keith Cirkel](https://avatars3.githubusercontent.com/u/118266?v=3&s=50)](https://github.com/keithamus)
+[![Lucas Fernandes da Costa](https://avatars3.githubusercontent.com/u/6868147?v=3&s=96)](https://github.com/lucasfcosta)
```
Comment (keithamus) [inline README.md]: This should end in `s=50`, not `s=96`

**Comment by lucasfcosta** [comment]
Thanks for noticing this! :sweat_smile:

**Comment by keithamus** [comment]
Welcome aboard!

**Comment by lucasfcosta** [comment]
Count me in, captain!

**Pull request merged**